definitions/schedule.py

```diff
@@ -33,5 +33,72 @@
                        data_dir="subnational/united-kingdom/collated",
                        dependencies=["united-kingdom",
                                      "united-kingdom-deaths",
-                                     "united-kingdom-admissions"])
+                                     "united-kingdom-admissions"]),
+            Dataset("belgium-full", data_dir="subnational/belgium-full/cases",
+                    max_wall_clock_minutes=7*24*60,
+                    timeout=3*24*60*60),
+            Dataset("regional-cases-full", data_dir="region/cases-full",
+                    max_wall_clock_minutes=7*24*60,
+                    timeout=3*24*60*60),
+            Dataset("regional-deaths-full", data_dir="region/deaths-full",
+                    max_wall_clock_minutes=7*24*60,
+                    timeout=3*24*60*60),
+            Dataset("united-kingdom-full",
+                    data_dir="subnational/united-kingdom-full/cases",
+                    max_wall_clock_minutes=7*24*60
+                    timeout=3*24*60*60),
+            Dataset("united-kingdom-full",
```

Comment:
You have misplaced -deaths and -admissions UK datasets and ended up with a triplicate of the base
united-kingdom-full
united-kingdom-deaths-full
united-kingdom-admissions-full

the same is true of the local